eahub/templates/eahub/profile.html

```diff
@@ -1,295 +1,302 @@
+                {% endthumbnail %}
+                {% if not profile.is_public %}
+                    <div class="alert alert-warning"
+                        role="alert"
+                        title="This profile is not public. Click on 'Edit' to change your settings."
+                    >
+                        <i class="fa fa-lock"></i>Private profile
+                    </div>
+                {% endif %}
+                {% if not profile.is_approved %}
+                    <div class="alert alert-warning"
+                        role="alert"
+                        title="This profile has not yet been approved, so other people can't see it yet."
+                    >
+                        <i class="fa fa-lock"></i> Awaiting approval
                     </div>
-                {% endfor %}
+                {% endif %}
             </div>
-        </div>
-    </div>
-{% endif %}
+            
+            <div class="col-xs-24 col-md-18">
+                <div class="prof__title row">
+                    <h1 class="col col-auto">{{ profile.name }}</h1>
+                
+                    <div class="col col-auto d-flex">
+                        {% if profile.facebook_url %}
+                            <a class="prof_social-link" href="{{ profile.facebook_url }}" rel="nofollow">
+                                <i class="fab fa-facebook"></i>
+                            </a>
+                        {% endif %}
+                        {% if profile.personal_website_url %}
+                            <a class="prof_social-link" href="{{ profile.personal_website_url }}" rel="nofollow">
+                                <i class="fa fa-link"></i>
+                            </a>
+                        {% endif %}
+                        {% if profile.linkedin_url %}
+                            <a class="prof_social-link" href="{{ profile.linkedin_url }}" rel="nofollow">
+                                <i class="fab fa-linkedin-in"></i>
+                            </a>
+                        {% endif %}
+                    </div>
 
+                    {% if request.user.id == profile.user_id %}
+                        <div class="prof__btns col col-24">
+                            <a class="btn btn-outline-primary btn-sm" href="{% url 'account_email' %}">
+                                <span><i class="fa fa-pencil-alt"></i> Change Email Address</span>
+                            </a>
+                            <a class="btn btn-outline-primary btn-sm" href="{% url 'account_change_password' %}">
+                                <span><i class="fa fa-pencil-alt"></i> Change Password</span>
+                            </a>
+                            <a class="btn btn-outline-primary btn-sm" href="{% url 'edit_profile' %}">
+                                <span><i class="fa fa-pencil-alt"></i> Edit</span>
+                            </a>
 
-{% if first_visit %}
```

Comment:
Why are you removing this? This was an ask from Catherine

Reply:
on the live site it isn't working at the moment

I advise replacing it with a different solution - passing parameters like that creates maintenance challenges because it goes against SOLID principles, hence it's easy to break and hard to notice the effect

Reply:
Agreed that we need different solution. Ticket: https://github.com/rtcharity/eahub.org/issues/1172